selfdrive/camerad/cameras/camera_qcom2.cc

```diff
@@ -581,93 +578,56 @@ static void camera_open(CameraState *s) {
   s->sensor_dev_handle = *sensor_dev_handle;
   LOGD("acquire sensor dev");
 
-  static struct cam_isp_resource isp_resource = {0};
-  isp_resource.resource_id = CAM_ISP_RES_ID_PORT;
-  isp_resource.length = sizeof(struct cam_isp_in_port_info) + sizeof(struct cam_isp_out_port_info)*(1-1);
-  isp_resource.handle_type = CAM_HANDLE_USER_POINTER;
-
-  struct cam_isp_in_port_info *in_port_info = (struct cam_isp_in_port_info *)malloc(isp_resource.length);
-  isp_resource.res_hdl = (uint64_t)in_port_info;
-
-  switch (s->camera_num) {
-    case 0:
-      in_port_info->res_type = CAM_ISP_IFE_IN_RES_PHY_0;
-      break;
-    case 1:
-      in_port_info->res_type = CAM_ISP_IFE_IN_RES_PHY_1;
-      break;
-    case 2:
-      in_port_info->res_type = CAM_ISP_IFE_IN_RES_PHY_2;
-      break;
-  }
-
-  in_port_info->lane_type = CAM_ISP_LANE_TYPE_DPHY;
-  in_port_info->lane_num = 4;
-  in_port_info->lane_cfg = 0x3210;
-
-  in_port_info->vc = 0x0;
-  //in_port_info->dt = 0x2C; //CSI_RAW12
-  //in_port_info->format = CAM_FORMAT_MIPI_RAW_12;
-
-  in_port_info->dt = 0x2B; //CSI_RAW10
-  in_port_info->format = CAM_FORMAT_MIPI_RAW_10;
-
-  in_port_info->test_pattern = 0x2; // 0x3?
-  in_port_info->usage_type = 0x0;
-
-  in_port_info->left_start = 0x0;
-  in_port_info->left_stop = FRAME_WIDTH - 1;
-  in_port_info->left_width = FRAME_WIDTH;
-
-  in_port_info->right_start = 0x0;
-  in_port_info->right_stop = FRAME_WIDTH - 1;
-  in_port_info->right_width = FRAME_WIDTH;
-
-  in_port_info->line_start = 0x0;
-  in_port_info->line_stop = FRAME_HEIGHT - 1;
-  in_port_info->height = FRAME_HEIGHT;
-
-  in_port_info->pixel_clk = 0x0;
```

Comment:
Can you bring over all the values that were set to zero. For documentation purposes it's better to make it explicit. Also keep the whitepsace of the original code.